lib/include/hse_ikvdb/cn_node_loc.h

```diff
@@ -121,10 +126,13 @@ cn_tree_max_depth(uint fbits)
  *
  */
 static inline u32
-nodes_in_level(u32 fanout_bits, u32 level)
+nodes_in_level(u32 fanout, u32 level)
 {
-    assert(fanout_bits * level < 32);
-    return (u32)1 << (fanout_bits * level);
+    uint fbits = ilog2(fanout - 1) + 1;
```

Comment:
Maybe add a macro or an inline function to convert fanout to fanout bits? I see this expression used in several places..

Reply:
This code will all go away soon, but I'll add the function..